### ISSUE TITLE
Path incorrect for godot-headers

### ISSUE THREAD

**Issue by SamuelEds** [touilleMan/godot-python#383]
I don't know if I'm missing something, but it needs the "godot-headers" submodule, however, it has some configuration, which I can't find, that it only points to the "godot_headers" path, but the correct one is "godot-headers".

At least, I cannot proceed with the settings as it is not creating the submodule with the `$ git submodule init` command.

- Python 3
- Pip 3
- Godot 3.5.1
- Git 2.39.1
- OS: Windows

![image](https://user-images.githubusercontent.com/52551197/215175500-1db8db46-b040-4ff7-8008-c26a5f1636ea.png)

**Comment by niklas2902** [comment]
Hi,
I tried to reproduced this error. But by using `git clone https://github.com/touilleMan/godot-python.git` and then calling `git submodule init`, I Can't reproduce this error.

Have you already found a way to fix this behavior?

My .gitmodules file looks like this:
```

[submodule "godot_headers"]
	path = godot_headers
	url = https://github.com/godotengine/godot_headers.git
	branch = 3.3

```
I hofe this helps

**Comment by SamuelEds** [comment]
Yes, thank you for answer. I remade the clone repository and submodule's process (I don´t know why, i guess that I maybe cloned a old repository) and worked. But, I get a new error (or better, several erros):

![image](https://user-images.githubusercontent.com/52551197/216399002-038dc4fd-5cf2-47ac-bd21-ad8c706617a2.png)
![image](https://user-images.githubusercontent.com/52551197/216399096-1af3b4a6-a3fa-4137-b211-7f749cd6be21.png)

I make the process in the virtual environment:

![image](https://user-images.githubusercontent.com/52551197/216399377-98631f0d-86e6-46ab-9c51-99cedce98779.png)

NOTE: The path: `./venv/bin/activate.bat` not worked. I used the path `venv\Scripts\activate.bat`

I remade any times, but I arrive in this part, i get theses errors. Am I forgetting anything? Any dependence?

**Comment by niklas2902** [comment]
I'm sorry to hear that you have another problem. Have you installed Microsoft Visual Studio with the corresponding C++ build tools?
That could be the problem: https://stackoverflow.com/a/49986365

**Comment by SamuelEds** [comment]
Yeah, i have installed the MSV C++:

![image](https://user-images.githubusercontent.com/52551197/217427024-0615270f-a3a5-4a1a-9137-6262e82a4f2f.png)

But I get continue with errors:

![image](https://user-images.githubusercontent.com/52551197/217427484-4e381ee0-e33e-42b6-8bad-141bdf41d7a9.png)

I re-installed everything, updated pip, checked the environment variables, tried searching the internet for something similar, but I didn't find anything useful. I do not know what else to do :/

**Comment by niklas2902** [comment]
Hm, ok. It seems to be about setuptools. Maybe you could just try to install it manually by calling `pip install setuptools` and this helps. I don't think, that the version specified in requirements.txt should matter too much

**Comment by SamuelEds** [comment]
I tried your recommendation but I keep getting the error kakaka. Anyway, I went to test some Script in Python on Godot with the plugin and it worked, which before just kept crashing, so I came here to see the repository. I think I just needed to update MSV C++.

But anyway, at least I don't have to worry about any errors here in the repository anymore. If I have another error I come here. Thank you very much friend for your patience!!
